health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewChecklist.js

```
@@ -18,53 +18,46 @@ const ViewChecklist = () => {
+        url: `/service-request/service/definition/v1/_search`,
+        body: {
+            ServiceDefinitionCriteria: {
+                "tenantId": tenantId,
+                "code": [serviceCode]
+            },
+            includeDeleted: true
+        },
+        config: {
+            select: (res) => {
+                if (res?.ServiceDefinitions?.[0]?.attributes) {
+                    let temp_data = res?.ServiceDefinitions?.[0]?.attributes
```

Biome

[error] This let declares a variable that is only assigned once.

'temp_data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)